Dockerfile

```diff
@@ -45,5 +45,3 @@ ENV PYTHONPATH="/app:/app/megaparse/sdk"
 
 COPY . .
 EXPOSE 8000
-
-CMD ["uvicorn", "megaparse.api.app:app", "--host", "0.0.0.0", "--port", "8000"]
```

Comment:
don't remove that or docker can't start unless provided explicit command

Reply:
Ok, but we don't have this in quivr-core for instance

Reply:
fixed